src/screens/CreatePost/elements/SheetExpiredPost.js

```diff
@@ -4,54 +4,53 @@ import {StyleSheet, Text, View} from 'react-native';
 import RBSheet from 'react-native-raw-bottom-sheet';
 
 import FlatListItem from '../../../components/FlatListItem';
+import {colors} from '../../../utils/colors';
 import {fonts} from '../../../utils/fonts';
-import {COLORS} from '../../../utils/theme';
 
-const SheetExpiredPost = ({refExpired, data, select, onSelect, isAnonimity = false}) => (
-  <RBSheet
-    ref={refExpired}
-    closeOnDragDown={true}
-    closeOnPressMask={true}
-    customStyles={{
-      container: styles.containerSheet,
-      draggableIcon: styles.draggableIcon
-    }}>
-    <View style={styles.container}>
-      <Text style={styles.header}>Post expiration after</Text>
-      {data.map((value, index) => (
-        <FlatListItem
-          key={index}
-          value={value.label}
-          index={index}
-          select={select}
-          onSelect={onSelect}
-          isAnonimity={isAnonimity}
-        />
-      ))}
-    </View>
-  </RBSheet>
-);
+const SheetExpiredPost = ({refExpired, data, select, onSelect}) => (
+    <RBSheet
+      ref={refExpired}
+      closeOnDragDown={true}
+      closeOnPressMask={true}
+      customStyles={{
+        container: styles.containerSheet,
+        draggableIcon: styles.draggableIcon,
+      }}>
+      <View style={styles.container}>
+        <Text style={styles.header}>Post expiration after</Text>
+        {data.map((value, index) => (
+          <FlatListItem
+            key={index}
+            value={value.label}
+            index={index}
+            select={select}
+            onSelect={onSelect}
+          />
+        ))}
+      </View>
+    </RBSheet>
+  );
 
 export default SheetExpiredPost;
 
 const styles = StyleSheet.create({
   container: {
     paddingTop: 20,
-    paddingBottom: 38
+    paddingBottom: 38,
   },
   header: {
-    color: COLORS.black,
+    color: colors.black,
     fontFamily: fonts.inter[600],
     fontSize: 18,
     fontWeight: 'bold',
-    paddingHorizontal: 20
+    paddingHorizontal: 20,
   },
   containerSheet: {
     borderTopRightRadius: 20,
     borderTopLeftRadius: 20,
-    height: 'auto'
+    height: 'auto',
   },
   draggableIcon: {
-    backgroundColor: COLORS.alto
-  }
+    backgroundColor: colors.alto,
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles for the `header` and `draggableIcon` components have been updated to use the new `colors` constant. As with the import statement, this change is fine as long as the `colors` constant has been updated to match the old `COLORS` constant. If not, this will cause a reference error.
```diff
-    color: COLORS.black,
+    color: colors.black,
...
-    backgroundColor: COLORS.alto
+    backgroundColor: colors.alto,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->